Frida/Script.swift

```diff
@@ -204,6 +204,43 @@ public class Script: NSObject, NSCopying {
             }
         }
 
+        if let dict = message as? [String: Any],
+            let type = dict["type"] as? String,
+            type == "send",
+            let payload = dict["payload"] as? [Any],
+            let kind = payload.first as? String,
+            kind == "frida:rpc",
+            let requestId = payload[1] as? Int,
+            let success = payload[2] as? String,
+            let script = connection.instance,
+            let callback = script.rpcCallbacks[requestId] {
+            
+            if success == "ok" {
+                var result: Any?
+                
+                if let data = data {
+                    result = data
+                } else if payload.count >= 4 {
+                    result = payload[3]
+                }
+                callback(.success(value: result))
+            } else {
+                let message: String
+                
+                if payload.count >= 3, let string = payload[2] as? String {
+                    message = string
+                } else if let string = payload[0] as? String {
+                    message = string
+                } else {
+                    message = "Failed to cast error message as string: \(payload)"
+                }
+                callback(.error(error: message))
```

Comment:
Perhaps a blank line before this line would improve readability by making it a little bit more symmetrical.